packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -57,8 +57,9 @@ export class ConnectionStateHandler {
         private readonly logger: ITelemetryLogger,
     ) {
         this.prevClientLeftTimer = new Timer(
-            // Default is 90 sec for which we are going to wait for its own "leave" message.
-            this.handler.maxClientLeaveWaitTime ?? 90000,
+            // Default is 5 min for which we are going to wait for its own "leave" message. This is same as
+            // the max time on server after which leave op is sent.
+            this.handler.maxClientLeaveWaitTime ?? 300000,
```

Comment:
@jatgarg -- Follow-up here - can we pull this value from a config in case we need to tweak it again for an incident?  Any other values in this area where we should do the same?

cc @anthony-murphy